Forcestyle literal flag added

BirgerMoell opened the pull request:
Added forcestyle literal flag to be able to force literal styling.
The merge request is dependent on the following merge request ins js-yaml.
https://github.com/nodeca/js-yaml/issues/464

MikeRalphson commented:
Thanks.

What is the effect of setting this new flag on the output?

Note: options should be defined using yargs syntax, and README would need updating.

BirgerMoell commented:
You can check the issue for more information.

With the flag added the input will stay in the first format instead of squishing to one line.
// With flag
INPUT:  abc:
  - name: xxx
    sample: |
      {
          "abc": 2,
          "def" : {
          "efg" : {
      	"ok" : 6666,
      	"nope" : "./yyy", 

// Without flag
OUTPUT:  abc:
  - name: xxx
    sample: "{\n    \"abc\": 2,\n    \"def\" : {\n    \"efg\" : {\n\t\"ok\" : 6666,\n\t\"nope\" : \"./yyy\",\n" 

I updated the code to use yargs for the flag and updated the comment.

MikeRalphson commented:
This project no-longer uses `js-yaml`, so I'm closing this for now. Please test the latest version (or wait on the next release) to see if the `yaml` module needs any further options supplying.
